pybikes/ecobici_ba.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2015, eskerda <ustroetz@gmail.com>
+# Distributed under the AGPL license, see LICENSE.txt
+
+from lxml import etree
+
+
+from pybikes.base import BikeShareSystem, BikeShareStation
+from pybikes.utils import PyBikesScraper
+
+
+FEED_URL = 'https://recursos-data.buenosaires.gob.ar/ckan2/ecobici/estado-ecobici.xml'
+
+
+class EcobiciBA(BikeShareSystem):
+    sync = True
+
+    meta = {
+        'system': 'ecobici_ba',
```

Comment:
This `system` does not represent anything inside the system, but the actual official name of the bike sharing system being used, if it applies.

Let's say Company A has a bike share system (shared between cities) called `Fancy Bikes™`. That would be what would go on system here.

On this case, I would say maybe `BA Ecobici`?